File: 231-power-of-two/power.py
class Solution:
    def isPowerOfTwo(self, n: int) -> bool:
        if n==1:
            return True
        exp = 0
        newN= n
        while newN >= 2:
            exp+=1
            newN = newN / 2
        return 2 ** exp == n
        

File: 231-power-of-two/power.js
/**
 * @param {number} n
 * @return {boolean}
 */
var isPowerOfTwo = function (n) {
    let exp = 0
    let newN = n
    while (newN >= 2) {
        exp++
        newN = newN / 2
    }
    return 2 ** exp === n
};
